WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/traffic/TrafficBarChartViewHolder.kt

```diff
@@ -178,6 +178,15 @@ class TrafficBarChartViewHolder(parent: ViewGroup) : BlockListItemViewHolder(
     private fun buildDataSet(context: Context, cut: List<BarEntry>): BarDataSet {
         val dataSet = BarDataSet(cut, "Data")
         dataSet.color = ContextCompat.getColor(context, R.color.blue_50)
+        dataSet.setGradientColor(
+            ContextCompat.getColor(
+                context,
+                R.color.blue_50
+            ), ContextCompat.getColor(
+                context,
+                R.color.blue_50
+            )
+        )
```

Comment:
This appears to be a bug from the chart library. Although we're updating `dataSet` of the `BarChart`, `BarChart`s `BarChartRenderer` is still using gradient colors from the previous chart.
I see a minor side effect of setting gradient colors again here. It causes the chart to render gradient color as a single color, an unnecessary process from the performance perspective.
May I suggest a simpler alternative solution? By setting the shader to null, we can prevent `BarChartRenderer`s from drawing gradient. 


```suggestion
        chart.renderer.paintRender.shader = null
        dataSet.color = ContextCompat.getColor(context, R.color.blue_50)
```

Reply:
Hi @irfano thanks for the suggestion! Yes, I agree that it's an issue with the library itself. I was concerned about the unnecessary process as well but noticed that `BarChartViewHolder` uses a similar workaround (lines 233-241 and 256-264): 

```        dataSet.setGradientColor(
            ContextCompat.getColor(
                context,
                R.color.stats_bar_chart_top
            ), ContextCompat.getColor(
                context,
                R.color.stats_bar_chart_top
            )
        )
```
I can go ahead and update the usage in `TrafficBarChartViewHolder`, but do you think it's a good idea to update these other occurrences as well?

Reply:
Yes, I think we can update it on both places. I tested my suggestion and didn't notice any issues.